package/ffnw-hoods/files/lib/ffnw/hoods/hoods.json

```diff
@@ -0,0 +1,64 @@
+[
+  {
+    "name": "default",
+    "bssid": "02:CA:FF:AA:BA:BF",
+    "defaulthood": true,
+    "servers": [
+      {
+        "host": "default02.entenhausen.de",
```

Comment:
Why are you using a different structure for the fastd configuration than the site.conf?